qiita_pet/test/rest/test_study.py

```diff
@@ -0,0 +1,167 @@
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from unittest import main
+
+from tornado.escape import json_decode
+
+from qiita_db.study import Study
+from qiita_pet.test.rest.test_base import RESTHandlerTestCase
+
+
+class StudyHandlerTests(RESTHandlerTestCase):
+    def test_get_valid(self):
+        exp = {u'title': u'Identification of the Microbiomes for Cannabis '
+                         u'Soils',
+               u'contacts': {'principal_investigator': [u'PIDude',
+                                                        u'Wash U',
+                                                        u'PI_dude@foo.bar'],
+                             'lab_person': [u'LabDude',
+                                            u'knight lab',
+                                            u'lab_dude@foo.bar']},
+               u'study_abstract':
+                   (u'This is a preliminary study to examine the '
+                    u'microbiota associated with the Cannabis plant. '
+                    u'Soils samples from the bulk soil, soil '
+                    u'associated with the roots, and the rhizosphere '
+                    u'were extracted and the DNA sequenced. Roots '
+                    u'from three independent plants of different '
+                    u'strains were examined. These roots were '
+                    u'obtained November 11, 2011 from plants that '
+                    u'had been harvested in the summer. Future '
+                    u'studies will attempt to analyze the soils and '
+                    u'rhizospheres from the same location at '
+                    u'different time points in the plant lifecycle.'),
+               u'study_description': (u'Analysis of the Cannabis Plant '
+                                      u'Microbiome'),
+               u'efo': [1],
+               u'study_alias': 'Cannabis Soils'}
+
+        response = self.get('/api/v1/study/1', headers=self.headers)
+        self.assertEqual(response.code, 200)
+        obs = json_decode(response.body)
+        self.assertEqual(obs, exp)
+
+    def test_get_invalid(self):
+        response = self.get('/api/v1/study/0', headers=self.headers)
+        self.assertEqual(response.code, 404)
+        self.assertEqual(json_decode(response.body),
+                         {'message': 'Study not found'})
+
+    def test_get_invalid_negative(self):
+        response = self.get('/api/v1/study/-1', headers=self.headers)
+        self.assertEqual(response.code, 404)
+        # not asserting the body content as this is not a valid URI according
+        # to the regex associating the handler to the webserver
+
+    def test_get_invalid_namespace(self):
+        response = self.get('/api/v1/study/1.11111', headers=self.headers)
+        self.assertEqual(response.code, 404)
+        # not asserting the body content as this is not a valid URI according
+
```

Comment:
I guess: "# to the regex associating the handler to the webserver" is missing ...